### ISSUE TITLE
IMS not registered e-sim

### ISSUE THREAD

**Issue by PontoZero23** [kyujin-cho/pixel-volte-patch#128]
I did the configuration procedures to activate the IMS it works in the physical chip, but in the virtual chip it continues as unregistered is this normal?! 
IMS Registered Physical Chip
IMS not registered E-Sim Chip..
App 1.2.4 
version Android 13 TQA.230505.002
Pixel 6
![IMS ](https://github.com/kyujin-cho/pixel-volte-patch/assets/129334377/e8023b36-44aa-42cc-b6a9-5ca05fb22d8d)

**Comment by arsradu** [comment]
Try this solution: https://github.com/kyujin-cho/pixel-volte-patch/discussions/7

**Comment by Buquiog** [comment]
It stopped working yesterday, 2023/05/10, after updating the Carrier Services app.  This application updates almost every week and whenever it updates, I delete its storage data to avoid problems, however, even deleting the app data, I am not able to make a voice call in LTE. 

If you want to exchange more information, my WhatsApp is +5567999764566

**Comment by arsradu** [comment]
I've never had any issues. Regardless of the carrier services app. Didn't bring anything, but also didn't need anything up.

**Comment by arsradu** [comment]
> in the case of command _#_#4363#_#_ the physical chip is registered and enables the functions and the e-sim chip is not registered but enables the functions, the connection works normally and even LTE+ is on my mobile data

I'm guessing you're talking about *#*#4636#*#*. And...if it works, i guess it could be just a display issue with that "unregistered". It's showing the same in my case. It could be normal though.

**Comment by PontoZero23** [comment]
yes, this is the code, I'm using it more because of doubts, and when an application update comes up, I'll test it, thanks for the tips and information

**Comment by fillwithjoy1** [comment]
> Hello! I made a discovery regarding the issue of not activating the IMS for the E-sim chip1 (ClaroBR) on my Pixel. The reason is that the carrier I use does not yet provide this technology for my region. So, I believe that when it is made available, it will work perfectly!

In that case even with pSIM it shouldn't matter - your carrier doesn't support IMS in general. (also do close the issue

**Comment by fillwithjoy1** [comment]
@kyujin-cho requesting to close this issue
